fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/derby/DerbyParameterNamesDAO.java

```diff
@@ -22,8 +22,8 @@
 public class DerbyParameterNamesDAO extends ParameterNameDAOImpl {
```

Comment:
Technically we are not use Parameter Name, we're using `code`... just something to think about.